twill11/hometask-6/lib/csv_writer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'csv'
+require_relative 'parser'
+
+# Writing info to csv file
+class CsvWriter
+  HEADERS = ['photo id', 'person real name', 'photo title', 'link to image'].freeze
+
+  attr_reader :data
+
+  def initialize(data)
+    @data = data
+  end
```

Comment:
Pass array of photos in the initializer.

```suggestion
  def initialize(photos)
    @photos = photos
  end
```